### PR TITLE
Arachne Update (Again)

### PR DESCRIPTION
**Pull request by Airomeda**
## Updates

Describe new patches added by your code, e.g.
- Apparel

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

**Review by albertbokor — APPROVED** [review]
LGTM!

File: Patches/Arachne/Apparel_Arachne.xml [hunk 1]
```diff
@@ -11,40 +11,368 @@
...
+				<value>
+					<Plasteel>60</Plasteel>
+					<DevilstrandCloth>20</DevilstrandCloth>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="Arachne_Power_Helmet"]/equippedStatOffsets</xpath>
+				<value>
+					<PsychicSensitivity>-0.2</PsychicSensitivity>
```
Comment (N7Huntsman) [inline Patches/Arachne/Apparel_Arachne.xml:330]: This is a duplicate tag.

File: Patches/Arachne/ApparelNew_Arachne.xml [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+				<value>
+					<Plasteel>50</Plasteel>
+					<DevilstrandCloth>15</DevilstrandCloth>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs/ThingDef[defName="Arachne_Scout_Helmet"]/equippedStatOffsets</xpath>
+				<value>
+					<PsychicSensitivity>-0.2</PsychicSensitivity>
```
Comment (N7Huntsman) [inline Patches/Arachne/ApparelNew_Arachne.xml:360]: This is also a duplicate tag.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
What's here generally looks good, but there are two occurrences of duplicate xml tags.

**Comment by Airomeda** [comment]
huhh guess it's overhead on my hand, I'll sort it quickly

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**